src/llmcompressor/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -64,75 +73,108 @@ def save_pretrained_wrapper(
             saving a model in dense format
             :param kwargs: additional kwargs to pass on to model.save_pretrained
             """
+            if is_fsdp_model(trainer.model):
```

Comment:
This function does too many things, would it be possible to break it out into smaller methods?

Reply:
+1 if we could break it out to help with readability

Reply:
okk